sdk/datafactory/arm-datafactory/CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
+  - Interface TeradataLinkedService has a new optional parameter maxRespSize
+  - Interface TeradataLinkedService has a new optional parameter portNumber
+  - Interface TeradataLinkedService has a new optional parameter sslMode
+  - Interface TeradataLinkedService has a new optional parameter useDataEncryption
+  - Interface TypeConversionSettings has a new optional parameter dateFormat
+  - Interface TypeConversionSettings has a new optional parameter timeFormat
+  - Type of parameter type of interface CopySink is changed from "DelimitedTextSink" | "JsonSink" | "OrcSink" | "RestSink" | "AzurePostgreSqlSink" | "AzureMySqlSink" | "AzureDatabricksDeltaLakeSink" | "WarehouseSink" | "SapCloudForCustomerSink" | "AzureQueueSink" | "AzureTableSink" | "AvroSink" | "ParquetSink" | "BinarySink" | "IcebergSink" | "BlobSink" | "FileSystemSink" | "DocumentDbCollectionSink" | "CosmosDbSqlApiSink" | "SqlSink" | "SqlServerSink" | "AzureSqlSink" | "SqlMISink" | "SqlDWSink" | "SnowflakeSink" | "SnowflakeV2Sink" | "OracleSink" | "AzureDataLakeStoreSink" | "AzureBlobFSSink" | "AzureSearchIndexSink" | "OdbcSink" | "InformixSink" | "MicrosoftAccessSink" | "DynamicsSink" | "DynamicsCrmSink" | "CommonDataServiceForAppsSink" | "AzureDataExplorerSink" | "SalesforceSink" | "SalesforceServiceCloudSink" | "MongoDbAtlasSink" | "MongoDbV2Sink" | "CosmosDbMongoDbApiSink" | "LakeHouseTableSink" | "SalesforceV2Sink" | "SalesforceServiceCloudV2Sink" to "DelimitedTextSink" | "JsonSink" | "OrcSink" | "RestSink" | "TeradataSink" | "AzurePostgreSqlSink" | "AzureMySqlSink" | "AzureDatabricksDeltaLakeSink" | "WarehouseSink" | "SapCloudForCustomerSink" | "AzureQueueSink" | "AzureTableSink" | "AvroSink" | "ParquetSink" | "BinarySink" | "IcebergSink" | "BlobSink" | "FileSystemSink" | "DocumentDbCollectionSink" | "CosmosDbSqlApiSink" | "SqlSink" | "SqlServerSink" | "AzureSqlSink" | "SqlMISink" | "SqlDWSink" | "SnowflakeSink" | "SnowflakeV2Sink" | "OracleSink" | "AzureDataLakeStoreSink" | "AzureBlobFSSink" | "AzureSearchIndexSink" | "OdbcSink" | "InformixSink" | "MicrosoftAccessSink" | "DynamicsSink" | "DynamicsCrmSink" | "CommonDataServiceForAppsSink" | "AzureDataExplorerSink" | "SalesforceSink" | "SalesforceServiceCloudSink" | "MongoDbAtlasSink" | "MongoDbV2Sink" | "CosmosDbMongoDbApiSink" | "LakeHouseTableSink" | "SalesforceV2Sink" | "SalesforceServiceCloudV2Sink"
+  - Type of parameter type of interface ImportSettings is changed from "AzureDatabricksDeltaLakeImportCommand" | "SnowflakeImportCopyCommand" to "TeradataImportCommand" | "AzureDatabricksDeltaLakeImportCommand" | "SnowflakeImportCopyCommand"
+  - Added Enum KnownAzurePostgreSqlWriteMethodEnum
+  - Added Enum KnownGreenplumAuthenticationType
+  - Added Enum KnownOracleAuthenticationType
+  - Added Enum KnownValueType
+
+### Breaking Changes
+
+  - Type of parameter type of interface Dataset is changed one of the values from "LakeHouseTable" to "LakehouseTable"
```

Comment:
Just to confirm these breakings are approved by ARM?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/32483